README.md

```diff
@@ -1,72 +1,86 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Todo list App
 
-> Description the project.
+
+> This branch (todo__list) We did it by contributing with my coding partner we did in this branch the followings 👌 :
+
+
+- [x] Setup project with webpack
+- [x] Create index.html using HTML Webpack Plugin
+- [x] Create index.js
+- [x] Add array of to do tasks 💯
+- [x] Iterate array and add dynamically to HTML
+- [x] Create style.css loaded by Webpack Style/CSS Loader
+- [x] Locate files in /src
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- Flexbox
+- JavaScript
+- Use the javascript objects and arrays to store the data for every project.
+- Webpack 
 
-## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
 
 
 ## Getting Started
 
 **This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
 ### Install
 
+- `git clone https://github.com/osamaashraf6/Todo-list-webpack`
+- `npm install`
+
 ### Usage
 
-### Run tests
+- `npm start` to start the app
+- You can add tasks in "Add to your list..."
+- You can edit tasks with double click in the task or click in the Ellipsis icon to enter at EDIT MODE.
+- You can delete tasks with click in the Ellipsis icon to enter at EDIT MODE and click in Trash icon.
+
+
 
-### Deployment
+
+## Getting Started
```

Comment:
- [ ] Your readme contains repeated "Getting Started" sections. It also has content that needs to be removed (the text between **) 👍 

Kindly double-check your readme after making changes so that you are sure it doesn't have this kind of mistakes anymore 😅

Reply:
- Thanks 🙏 for reviewing my pull request I appreciate your review 💯 
### Thanks again for reviewing my PR  and have a good day😊